xbmc/utils/DiscsUtils.cpp

```diff
@@ -18,6 +20,14 @@
 #include "filesystem/BlurayDirectory.h"
 #endif
 
+#include "filesystem/File.h"
+
+#include <algorithm>
+#include <array>
+#include <ranges>
```

Comment:
The `<ranges>` header isn't needed, `std::ranges::any_of` is in `<algorithm>`.

Reply:
> `std::ranges::any_of` is in `<algorithm>`.

Confusing...